disktools.py

```diff
@@ -1267,3 +1267,76 @@ def getMdNodes():
     except IOError:
         pass
     return nodes
+
+
+class DeviceMounter:
+    class Mount:
+        def __init__(self, dev, mountpoint, options=None, fstype=None):
+            self.dev = dev
+            self.mountpoint = mountpoint
+            self.options = options
+            self.fstype = fstype
+
+        def __repr__(self):
+            return "Mount(%s, %s, %s, %s)" % (self.dev, self.mountpoint, self.fstype, self.options)
+
+        def mount(self):
+            util.mount(self.dev, self.mountpoint, self.options, self.fstype)
```

Comment:
With the previous shell script errors were just ignored. Should we be consistent and ignore and mount/umount errors?

Reply:
I think ignoring errors would be the wrong thing to do, even if it is a slight change in behaviour.

Reply:
Based on comment marking as resolved